modules/core/src/main/scala/com.snowplowanalytics.iglu/client/validator/CirceValidator.scala

```diff
@@ -88,25 +88,26 @@ object CirceValidator extends Validator[Json] {
     V4SchemaInstance.getSchema(new ObjectMapper().readTree(MetaSchemas.JsonSchemaV4Text))
 
   def validate(data: Json, schema: Json): Either[ValidatorError, Unit] = {
-    val jacksonJson = circeToJackson(schema)
+    val jacksonJson = circeToJackson(schema, Int.MaxValue)
```

Comment:
MaxValue for backwards compatibility.

This method is only ever used by `Client`, and I have plans to deprecate the old Client.  In future we should support only `IgluCirceClient` which is newer and better.